app/assets/javascripts/activeadmin_addons/inputs/nested-select.js

```diff
@@ -151,6 +151,9 @@ var initializer = function() {
 
       if (!!parent) {
         var parentSelectorId = '#' + model + '_' + parent;
+        if (!$(parentSelectorId).length) {
+            parentSelectorId = $(container).find('*[id*='+ parent +']')[0];
```

Comment:
Expected indentation of 10 spaces but found 12  indent<br>Infix operators must be spaced                  space-infix-ops